### PR TITLE
Remove name from source files and update readme

### PR DESCRIPTION
**Pull request by GjjvdBurg**

### CODE REVIEW

File: signal2html/__main__.py [hunk 1]
```diff
@@ -2,9 +2,7 @@
 
 """Command line interface
 
-Author: G.J.J. van den Burg
 License: See LICENSE file.
-Copyright: 2020, G.J.J. van den Burg
```
Comment (ericthegrey) [inline signal2html/__main__.py:7]: Perhaps the license file should mention '... and contributors'? I don't know if that's typical.

Comment (GjjvdBurg) [reply signal2html/__main__.py:7]: Good catch! What do you think about [allcontributors](https://allcontributors.org/)?

Comment (ericthegrey) [reply signal2html/__main__.py:7]: Sure if you want, I didn't know about that tool.

Comment (GjjvdBurg) [reply signal2html/__main__.py:7]: Let's leave it for now then, we can always add it later.

**Pull request merged**